agent/agent-api-public-deprecated.yaml

```diff
@@ -197,6 +197,14 @@ paths:
           in: query
           required: false
           type: boolean
+        - name: failOnDatafeedConnectivity
+          description: |
+            Defines the HealthCheck status code response whenever the Datafeed2 connectivity fails. When set to "true",
+            in case of Datafeed connectivity failure, the response status code will be 503; otherwise, it will be 200.
+            Default value is "false".
+          in: query
+          required: false
+          type: boolean
```

Comment:
Weird, I thought I actually removed this param from this endpoint. Let me check

Reply:
Ok, still here but not used in the implementation as we finally decided to ignore DF2 health until having it deployed in prod. We can let it as it is 👍